lib/packBuffers.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+
+var byteLengthForComponentType = require('./byteLengthForComponentType');
+var getAccessorByteStride = require('./getAccessorByteStride');
+var numberOfComponentsForType = require('./numberOfComponentsForType');
+
+module.exports = packBuffers;
+
+/**
+ * Repacks the accessed buffer data into contiguous chunks.
+ * Also has the effect of un-interleaving interleaved accessors.
```

Comment:
I don't think so; the un-interleaving is just a by-product of packBuffers copying the data into the smallest space possible. 

I could write a separate function to do un-interleaving without removing unused data, but it wouldn't make this one simpler and I can't think of a case where you would want to intentionally keep unused data.